plugins/src/main/java/opengrok/auth/plugin/ldap/LdapServer.java

```diff
@@ -134,7 +141,7 @@ public synchronized boolean isWorking() {
      * @return the new connection or null
      */
     private synchronized LdapContext connect() {
-        LOGGER.log(Level.INFO, "Server {0} connecting", this.url);
+        LOGGER.log(Level.INFO, "Connecting to LDAP server {0} ", this.toString());
```

Comment:
Nit: is `toString()` superfluous here and below (and if not, should these logging computations be short-circuited by convention with e.g. `if (LOGGER.isLoggable(Level.INFO))`)?